src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -406,6 +469,13 @@ private SuggestedActionSet OrganizeRefactorings(Workspace workspace, CodeRefacto
                 }
 
                 var selectedSpan = selectedSpans[0];
+
+                // We only support refactorings when selected span intersects with the span that the light bulb is asking for.
+                if (!selectedSpan.IntersectsWith(range))
```

Comment:
no, it should translate. if you see code below, it does translate to make sure everything is working on same snapshot. caller of this method should translate the span to expected snapshot.